tools/ci_build/build.py

```diff
@@ -12,7 +12,7 @@
 import hashlib
 from logger import get_logger
 from amd_hipify import amd_hipify
-
+from distutils.version import StrictVersion
```

Comment:
Can you only do this change for mac? Because not every one want to build onnxruntime's python extension. They may don't have distutils.